lnrpc/lightning.proto

```diff
@@ -1544,6 +1547,11 @@ message ListChannelsRequest {
     empty, all channels will be returned.
     */
     bytes peer = 5;
+
+    // Informs the server if the peer alias lookup per channel should be
+    // enabled. It is turned off by default in order to avoid degradation of
+    // performance of existing clients.
```

Comment:
nit: "for" existing clients? Since the client itself doesn't degrade in performance :)